app/models/spree/line_item_decorator.rb

```diff
@@ -4,6 +4,9 @@
   include OpenFoodNetwork::VariantAndLineItemNaming
   has_and_belongs_to_many :option_values, join_table: 'spree_option_values_line_items', class_name: 'Spree::OptionValue'
 
+  # Redefining here to add the inverse_of option
+  belongs_to :order, :class_name => "Spree::Order", inverse_of: :line_items
```

Comment:
I think it'd be nicer to have consistent hash syntax here (ie. `class_name: 'Spree::Order'`).